project-fortis-pipeline/ops/create-tags.sh

```diff
@@ -4,7 +4,18 @@ readonly k8resource_group="${1}"
 readonly fortis_interface_host="${2}"
 readonly site_name="${3}"
 readonly graphql_service_host="${4}"
+readonly dns_host_name="${5}"
+readonly dns_host_ip="${6}"
+readonly mx_record_entry="${7}"
 
 az group update --name "${k8resource_group}" --set tags.FORTIS_INTERFACE_URL="${fortis_interface_host}/index.html#/site/${site_name}"
```

Comment:
Is the `fortis_interface_host` being set to https when the backend is protected via TLS? We don't want to be pointing our users to a http frontend when the backend is https.

Reply:
`fortis_interface_host` is tied to the use `aad_client`, so not sure what the proper logic to impose here is.  Note (and below):

https://github.com/jmspring/project-fortis/blob/6bff4da10d9b302b21f8e93d7d89d7cf9b4c2db9/project-fortis-pipeline/ops/create-cluster.sh#L30

There are resources tied to http vs https.  Why wouldn't anything on blob store be `https` always and not http.